website/src/docs/hotchocolate/defining-a-schema/index.md

```diff
@@ -1,510 +1,57 @@
 
-</ExampleTabs.Annotation>
-<ExampleTabs.Code>
+- [_Object types_](/docs/hotchocolate/defining-a-schema/object-types) contain fields and describe our entities.<br />[Operations](#Operations) for example are nothing more than simple object types.
 
-```csharp
-// Query.cs
-public class Query
-{
-    public Task<Book> GetBookAsync(string id)
-    {
-        // Omitted code for brevity
-    }
-}
+- [Scalars](/docs/hotchocolate/defining-a-schema/scalars) are the primitives of our GraphQL schema: `String`, `Int`, etc.<br />We can also define custom scalars to more precisely describe our business domain.
 
-// QueryType.cs
-public class QueryType : ObjectType<Query>
-{
-    protected override void Configure(IObjectTypeDescriptor<Query> descriptor)
-    {
-        descriptor
-            .Field(f => f.GetBook(default))
-            .Type<BookType>();
-    }
-}
+Besides those there are also [enums](/docs/hotchocolate/defining-a-schema/enums), [interfaces](/docs/hotchocolate/defining-a-schema/interfaces) and [unions](/docs/hotchocolate/defining-a-schema/unions).
 
-// Book.cs
-public class Book
-{
-    public string Title { get; set; }
+## Arguments
 
-    public string Author { get; set; }
-}
+We can pass [arguments](/docs/hotchocolate/defining-a-schema/arguments) to individual fields on an object type and access their values inside the field's resolver.
 
-// BookType.cs
-public class BookType : ObjectType<Book>
-{
-    protected override void Configure(IObjectTypeDescriptor<Query> descriptor)
-    {
-        descriptor
-            .Field(f => f.Title)
-            .Type<StringType>();
+Nested (object) types can also be used as arguments by declaring so called [input object types](/docs/hotchocolate/defining-a-schema/input-object-types). These are most commonly used when passing a payload to a mutation.
 
-        descriptor
-            .Field(f => f.Author)
-            .Type<StringType>();
-    }
-}
+## Type Modifiers
 
+Besides regular types, like scalars and object types, there are also _type modifiers_.
 
-// Startup.cs
-public class Startup
-{
-    public void ConfigureServices(IServiceCollection services)
-    {
-        services
-            .AddRouting()
-            .AddGraphQLServer()
-            .AddQueryType<QueryType>();
-    }
+A [Non-Null](/docs/hotchocolate/defining-a-schema/non-null) field for example indicates that a client can always expect a non-null value to be returned from the field.
 
-    // Omitted code for brevity
-}
-```
+[List](/docs/hotchocolate/defining-a-schema/lists) fields indicate to a client that the field will return an ordered list in the specified shape.
 
-</ExampleTabs.Code>
-<ExampleTabs.Schema>
+## Extending Types
 
-```csharp
-// Query.cs
-public class Query
-{
-    public Task<Book> GetBookAsync(string id)
-    {
-        // Omitted code for brevity
-    }
-}
+Hot Chocolate allows us to [extend existing types](/docs/hotchocolate/defining-a-schema/extending-types), helping us keep our code organized.
```

Comment:
Maybe something like

```suggestion
Hot Chocolate allows us to [extend existing types](/docs/hotchocolate/defining-a-schema/extending-types), helping us keep our code organized. Rather than adding more and more fields to the Query type in the same class for instance, you can _extend_ the Query type with a new field from another location in your codebase where that field logically should live.
```